HousingSearchListener/V1/Domain/Person/Person.cs

```diff
@@ -17,5 +17,20 @@ public class Person
         public string DateOfBirth { get; set; }
         public List<string> PersonType { get; set; }
         public List<Tenure> Tenures { get; set; }
+
+        public string FullName => FormatFullName();
+
+        private string FormatFullName()
+        {
+            string firstName = FormatNamePart(FirstName);
+            string middleName = FormatNamePart(MiddleName);
+            string surname = FormatNamePart(Surname);
+            return $"{Title}{firstName}{middleName}{surname}";
+        }
+
+        private static string FormatNamePart(string part)
+        {
+            return string.IsNullOrEmpty(part) ? string.Empty : $" {part}";
+        }
```

Comment:
This should definitely go in the person shared nuget package.

Reply:
I will do as part of my tech debt ticket.